### ISSUE TITLE
Check if we should implement merging of FN, FNDA, FNF, FNH, BRF, BRH, LF, LH

### ISSUE THREAD

**Issue by mweibel** [mweibel/lcov-result-merger#14]
See [man page](http://ltp.sourceforge.net/coverage/lcov/geninfo.1.php) for info.

**Comment by jparra5** [comment]
@mweibel I can provide an lcov file:

[lcov.info.zip](https://github.com/mweibel/lcov-result-merger/files/1700708/lcov.info.zip)

I just noticed that lcov-result-merger was stripping the LF & LH fields when I merged two files together.

**Comment by mweibel** [comment]
Thanks @jparra5. 
I'll look into it as soon as I find some time. Obviously I'd be open for a PR too :)

**Comment by mweibel** [comment]
Sorry, forgot about this. It's back on my radar :)

**Comment by cellog** [comment]
is this project dead now?

**Comment by sbhark** [comment]
Hi there, 
I was just wondering if there is progress in the implementation of this issue? 
Thanks,

**Comment by sbhark** [comment]
Alright, will do.

**Comment by floric** [comment]
Yeah, LF seems to be important. I'd like to use the merger together with `lcov-summary`. But this currently fails with `NaNs` because of the missing LF... :O

**Comment by elashpear** [comment]
I am working around this now by using official `lcov` binary to merge reports as described at https://wiki.documentfoundation.org/Development/Lcov#Combine_lcov_tracefiles

**Comment by Gideon-Felt** [comment]
> I am working around this now by using official `lcov` binary to merge reports as described at https://wiki.documentfoundation.org/Development/Lcov#Combine_lcov_tracefiles

This was the best solution for me as well.
